packages/test/test-end-to-end-tests/src/test/serializeAfterFailedAttach.spec.ts

```diff
@@ -95,171 +94,191 @@ describeCompat(
 			};
 		};
 
-		it("Can serialize detached container", async () => {
-			const provider = getTestObjectProvider();
-			const documentId = createDocumentId();
-			const request = provider.driver.createCreateNewRequest(documentId);
-			const loader = createTestLoader(provider);
-
-			const { container } = await createDetachedContainerAndGetEntryPoint(loader, request);
+		for (const attachAfterRehydrate of [true, false]) {
+			it(`Can serialize detached container. attachAfterRehydrate: ${attachAfterRehydrate}`, async () => {
```

Comment:
pre-existing, but "detached" in all of these test case names seems a bit weird: they all use the helper which puts the container in an attaching state before serializing.

anyway, I think it's worth including in the test case name the fact that this one has *no* operations after the failed attach. Makes its use as a simpler variant of below tests more obvious.